src/index.html

```diff
@@ -13,6 +13,131 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), class names should not be tied to the content, as they should be reusable. Consider using a more generic name instead of `calendar--month-length-31`.